internal/pkg/deploy/upload/asset/asset.go

```diff
@@ -43,6 +44,7 @@ type asset struct {
 
 	ArtifactBucketPath string `json:"path"`
 	ServiceBucketPath  string `json:"destPath"`
+	ContentType        string `json:"contentType"`
```

Comment:
nit: do you think it's worthwhile doing `,omitempty` for these tags to just have a file that's slightly smaller if the field is empty?

Reply:
I thought about this but the file itself is relatively small and maybe it's good to keep each element in the array consistent to have the same key set?